src/vendor/vscode-ocaml-platform/src-bindings/node/node.ml

```diff
@@ -198,6 +198,8 @@ module Net = struct
   module Socket = struct
     include Class.Make ()
 
+    let to_ojs (t : t) : Ojs.t = (t :> Ojs.t)
+
```

Comment:
Note: there's already
```ocaml
  val t_of_js : Ojs.t -> t
  val t_to_js : t -> Ojs.t
```
in this module.

Reply:
Sorry I though that was in the actual public signature, but it's not.

Ideally arguments to `Vscode_languageclient.StreamInfo.create` would be of `Node.Net.Socket.t`. Doing this may require adding both `t_of_js` and `t_to_js` in the public signature of `Node.Net.Socket`; other JS stubs libraries also seem to expose these magic bindings.

See https://github.com/nberth/superbol-studio-oss/commit/f6434e3a180bce505bf07b5f43c1797c2527b6eb